### DISCUSSION TITLE
Replace "Browser" with Firefox

### DISCUSSION THREAD

**Discussion by dylanmtaylor** [linuxdeepin/developer-center#7756] [Features Request & Ideas | 特性请求 & 头脑风暴]
Deepin seems to ship with a custom browser that is a fork of Chromium. Rather than repeating this effort, why not ship Firefox?

**Comment by c61292558** [comment]
firefox doesn't seem to have a great deal of localization support?Or is it a copyright issue?

**Comment by BLumia** [comment]
The deepin distro doesn't have plan to replace it's in-house chromium-fork to Firefox due to various reason, you can replace it to Firefox after the installation if you like :)
